.github/workflows/tests.yml

```diff
@@ -68,6 +68,12 @@ jobs:
       - name: Setup MSVC
         uses: egor-tensin/vs-shell@9a932a62d05192eae18ca370155cf877eecc2202 # v2.1
         if: ${{ matrix.target_os == 'windows' }}
+      - name: install python3-requests
+        run: pip3 install requests
```

Comment:
Could you pin the hash and version?

Reply:
Fixed, but without hashes. `--hash` is only available through `requirements.txt` and if I put it for one package, it is required for all deps and sub-deps, so only proceeding with version pining.